.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  schedule:
+      # once per day
+    - cron: 0 0 * * *
+  push:
+    branches:
+      - development
+      - master
+  pull_request:
+    branches:
+      - development
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run test-ci
+      env:
+           CI: true
```

Comment:
Any reason why this one is indented with 5 spaces instead of 2?

Reply:
I believe that's just the way it has to be formatted

Reply:
Well as long as it works...